config/init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE DATABASE IF NOT EXISTS directus;
+USE directus;
+
+CREATE TABLE IF NOT EXISTS adopted_probes (
+  id INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+  user_created CHAR(36),
+  date_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  user_updated CHAR(36),
+  date_updated TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  userId VARCHAR(255) NOT NULL,
+  ip VARCHAR(255) NOT NULL,
+  uuid VARCHAR(255),
+  lastSyncDate DATE
+);
+INSERT IGNORE INTO adopted_probes (id, userId, ip) VALUES ('1', '6191378', '79.205.97.254');
+
+CREATE USER IF NOT EXISTS 'directus';
+GRANT ALL PRIVILEGES ON `directus`.* TO 'directus';
+FLUSH PRIVILEGES;
```

Comment:
This is dev only and in production, we rely on directus migrations, right?

Reply:
yes, directus uses `directus` user so it will be in place in production. maybe we should also have a specific user for gp api?

Reply:
User can be the same.